lib/public/models/OverviewModel.js

```diff
@@ -168,6 +170,23 @@ export class OverviewPageModel extends Observable {
         return this._pagination;
     }
 
+    /**
+     * Get value stating whether table should be horizontally scrollable
+     * @return {boolean} whether display is scrollable
+     */
+    get displayXScrollable() {
```

Comment:
This is my personal feeling, I would rather name it "isHorizontalScrollEnabled" and "setHorizontalScrollEnabled"

Reply:
applied